### PR TITLE
Readme to change image, and fix installation commands

### PR DESCRIPTION
**Pull request by davidkartuzinski**
#### Description of what you did:

Updated the main image in the Readme to show Groups in the best way.  Also shows new UI.

Fixed some grammar issues and spellings.

Replaced the old install command with the new recommend Yarn command with the npx fallback. As per discussion, I moved the commands above the requirements to ensure they are seen fast. 

Other changes are minor.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,30 +25,54 @@
...
+- **Customizable**. You can quickly build your logic by fully customizing APIs, routes, or plugins to fit your needs perfectly.
 
 ## Getting Started
 
 <a href="https://strapi.io/documentation/3.0.0-beta.x/getting-started/quick-start.html" target="_blank">Read the Getting Started tutorial</a> or follow the steps below:
 
-#### 🖐 Requirements
+### ⏳ Installation
+
+Install Strapi with this **Quickstart** command to install and create a Strapi project instantly:
```
Comment (derrickmehaffy) [inline README.md]: Add a note that this uses SQLite

File: README.md [hunk 2]
```diff
@@ -130,17 +129,18 @@ Please read our [Contributing Guide](./CONTRIBUTING.md) before submitting a Pull
 
 ## Support
 
-For more information on the upcoming version, please take a look to our [ROADMAP](https://github.com/strapi/strapi/projects).
+For more information on the upcoming version, please take a look at our [ROADMAP](https://github.com/strapi/strapi/projects).
```
Comment (derrickmehaffy) [inline README.md]: Should move the Roadmap link to product board

File: README.md (hunk 1 above)
Comment (davidkartuzinski) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (davidkartuzinski) [reply README.md]: This sentence is actually totally unnecessary.  Later we say practically the same thing but better. So I deleted this instance.

File: README.md (hunk 1 above)
Comment (soupette) [reply README.md]: What about :

Get started instantly with the quickstart command to create a Strapi project.

(So we avoid the create repetition)

Comment (davidkartuzinski) [reply README.md]: Thanks @soupette - I agree that the repetition of the word **"install"** is a bit awkward. (I think you're referring to this.) But you will see I only made the following change:

**Install Strapi with this **Quickstart** command to create a Strapi project instantly:**

As it still reads a better than rearranging the subject of the sentence (as in your example).

**Pull request merged**